andela/travel_tool_front
# #161051781 User can delete a request with 'open' status

#### What does this PR do?
This PR enables a requester to delete a request, their manager is then notified about the action carried out.

#### Description of Task to be completed?
-Travel requester should be able to delete only open requests
- There should be a pop up modal asking the user if they are sure they want to delete the travel request
- The request should be deleted from Manager's approval page
- The manager should get an in-app notification to this effect

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_back.git`
- Enter into the project directory - `cd travel_tool_back`
- checkout the branch `ft-delete-requests-161051781`
- Start the server - `make start`
- Navigate to [travela](http://travela-local.andela.com:3000/#) in your browser and log in
- Login on your browser and grab your token from the application tab in Developers tab
- Click on the ellipsis icon for an open request, you should see a drop-down with an option to delete
- Click on the delete button, confirm to delete the request

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
[161051781](https://www.pivotaltracker.com/story/show/161051781)

#### Screenshots (if appropriate)


#### Questions:
